sdk/postgresql/Microsoft.Azure.Management.PostgreSQL/src/Microsoft.Azure.Management.PostgreSQL.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.PostgreSQL</PackageId>
 		<Description>Provides developers with libraries for the Azure Database for PostgreSQL under Azure Resource manager to deploy an instance of PostgreSQL. Create, Update, Delete, Restore, and more.</Description>
-		<Version>0.9.0-preview</Version>
+		<Version>0.9.5-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.PostgreSQL</AssemblyName>
 		<PackageTags>PostgreSQL;Azure Database for PostgreSQL;</PackageTags>
 		<PackageReleaseNotes>
```

Comment:
Please update with more detailed release note if you have them

Reply:
updated